php/libraries/Utility.class.inc

```diff
@@ -587,54 +587,6 @@ class Utility
         return $instruments;
     }
 
-    /**
-     * Get a list of all instruments which have scoring implemented.
-     * Used in statistics module.
-     *
-     * @return array non-associative array of test_names
-     */
-    static function getScoredInstruments(): array
-    {
-        $All     = \Utility::getAllInstruments();
-        $Scored  = array();
-        $Nothing = array();
-        foreach (array_keys($All) as $Instrument) {
-            if (\Utility::getScoreColsForInstrument($Instrument) != $Nothing) {
-                $Scored[] = $Instrument;
-            }
-        }
-        return $Scored;
-    }
-
-    /**
-     * Return the score columns which exist for a given instrument.
-     *
-     * @param string $inst The test_name of the instrument for which you would
-     *                     like to know the score columns
-     *
-     * @return array (non-associative) of scored columns for the instrument given
-     */
-    static function getScoreColsForInstrument(string $inst): array
-    {
-        include_once "NDB_BVL_Instrument.class.inc";
-        $i = NDB_BVL_Instrument::factory($inst, '', '', true);
-        if (isset($i->scoreLabels)) {
```

Comment:
The `NDB_BVL_Instrument` object does not have the class variables `scoreLabels`, `scoreColumns` or `scores` so this function and the one above were removed.

Reply:
specific instrument instances in projects have them set.

Reply:
(note: the above comment doesn't mean you can't remove it.. I think it was for the same place as getScoredInstruments which doesn't exist anymore.)

Reply:
Good to know